test/utility/tiered.c

```diff
@@ -64,12 +64,11 @@ testutil_tiered_sleep(TEST_OPTS *opts, WT_SESSION *session, uint64_t seconds, bo
     uint64_t now, wake_time;
     bool do_flush;
 
-    WT_ASSERT((WT_SESSION_IMPL *)session, do_flush_tier != NULL);
-
     now = testutil_time_us(session);
     wake_time = now + WT_MILLION * seconds;
     do_flush = false;
-    if (opts->tiered_flush_next_us != 0 && opts->tiered_flush_next_us < wake_time) {
+    if (do_flush_tier != NULL && opts->tiered_flush_next_us != 0 &&
```

Comment:
If `do_flush_tier` *is* NULL what will happen later on line 75?
The `do_flush_tier` var looks like a signalling var (guess), what does it mean if it is NULL?

Reply:
This is a revert so better to leave it as is and then merge fixes once the original ticket is re-done.

Reply:
Ah my mistake -- did not realize we reviewed reverts.